<launch>
    <arg name="config_name" default="tradr_sys_teleop_config.yaml" />
    <remap from="/tradr/joy/cmd_vel" to="/cmd_vel"/>
    
    <node pkg="joy" type="joy_node" name="joy_node" output="screen"> 
	<param name="default_trig_val" type="bool" value="true" />
    </node>

    <node pkg="augmented_robot_trackers" type="tradr_sys_teleop.py" name="tradr_teleop" args="$(arg config_name)" output="screen" />
</launch>
